Sources/Subs.php

```diff
@@ -2101,23 +2123,16 @@ function parse_bbc($message, $smileys = true, $cache_id = '', $parse_tags = arra
 			// This is long, but it makes things much easier and cleaner.
 			if (!empty($possible['parameters']))
 			{
-				$preg = array();
+				// START: Dougiefresh's faster parse_bbc function
```

Comment:
We can't remove it unless it's merged into SMF, so you'll need to submit another commit to remove it.